src/WhiteLibrary/__init__.py

```diff
@@ -151,6 +151,7 @@ def __init__(self, screenshot_dir=None):
         self.screenshooter = None
         self.ROBOT_LIBRARY_LISTENER = self  # pylint: disable=invalid-name
         self.screenshots_enabled = True
+        self._attach_application_timeout = 30000
```

Comment:
Default value could be 0, so that the behavior of the kw would not change if the timeout value is not changed.